compendium/DeclarativeServices/test/gtest/TestFactoryTarget.cpp

```diff
@@ -0,0 +1,135 @@
+/*=============================================================================
+
+  Library: CppMicroServices
+
+  Copyright (c) The CppMicroServices developers. See the COPYRIGHT
+  file at the top-level directory of this distribution and at
+  https://github.com/CppMicroServices/CppMicroServices/COPYRIGHT .
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  =============================================================================*/
+
+#include "TestFixture.hpp"
+#include "TestInterfaces/Interfaces.hpp"
+#include "cppmicroservices/LDAPFilter.h"
+#include "cppmicroservices/LDAPProp.h"
+#include "cppmicroservices/servicecomponent/ComponentConstants.hpp"
+#include "gtest/gtest.h"
+
+namespace test
+{
+
+    class tFactoryTarget : public tGenericDSSuite
```

Comment:
derive from `tGenericDSAndCASuite` instead so you don't have to re-write `SetUp` and `TearDown`

Reply:
tGenericDSAndCASuite is part of the usConfigurationAdminTests project (TestFixtures.hpp). I didn't think we were allowed to use include files from another project.

Reply:
Added a tGenericDSAndCASuite which is derived from tGenericDSSuite to TestFixture.hpp